### ISSUE TITLE
JOSS reviewer comments

### ISSUE THREAD

**Issue by sachsmc** [ropensci/jagstargets#26]
Hi @wlandau , 

Thanks for the nice package, it is good timing for me to test out as I am currently working on a project using jags. Here come a few minor suggestions on the write up for JOSS: 

- [ ] First sentence: suggest rephrasing as "Researchers(/statisticians/people) performing Bayesian analyses regularly experiment with models to ..."
- [ ] Statement of need, general comment: I like what you have so far, but consider going further by additionally explaining something like: "the targets package helps researchers manage the complexity of multiple models, data, etc., but it lacks specific features for dealing with models specified in jags ... "
- [ ] Last paragraph line 33: "Bayesian statisticians" -> "Bayesian data analysis"
- [ ] The "State of the field" description is lacking. If I didn't want to use jagstargets (but still jags), what are my options? I suppose I could write some functional wrappers around the rjags calls and pass them to targets, what would be the disadvantage of doing that aside from the effort? 

In general, I suggest avoiding the phrase "Bayesian statisticians" in the documentation as it can be divisive. It is on in the first paragraph of the README as well. It is nit-picky, but one need not identify as a Bayesian statistician to do Bayesian data analyses. Actually now that I think about it you don't even need to be a statistician. 

Best regards,

Michael

**Comment by wlandau** [comment]
Thanks for your feedback, @sachsmc. I believe I addressed your first three comments in https://github.com/ropensci/jagstargets/commit/12da6316beae796ee13abb75a5955183ee9502a1, https://github.com/ropensci/jagstargets/commit/595f0cd20d51e4f3a9a8497bc17f3252eb06f537, and https://github.com/ropensci/jagstargets/commit/2c20af05d1ea6d0108046fcc23de60f821817184. Third paragraph of the paper is about your last comment:

> These pipelines, which can be visualized and executed using [`targets`](https://docs.ropensci.org/targets/) [@targets], support key features that increase the efficiency and reproducibility of Bayesian workflows. The steps of a pipeline are automatically orchestrated using optional distributed computing, and up-to-date tasks are automatically skipped if the upstream code and data did not change since the last run. Thus, researchers can quickly iterate on Bayesian workflows while maintaining agreement between the results and the underlying code, models, and datasets.

In other words, by leveraging `targets`, `jagstargets` brings powerful capabilities to Bayesian workflows that are over and above what in an ordinary function-based JAGS workflow:

1. Automatically skip computation (e.g. MCMC, post-processing, data generation) that is already up to date with the underlying code and upstream data. This is crucial, it routinely saves hours of runtime. In fact, I started working on the general problem of Make-like declarative workflows in R because of my experience with a non-pipeline workflow in grad school. I had a function-based workflow that ran about 20 models, each of which took 4 hours. Every time I changed the code I needed to rerun everything to be sure my results were up to date, and the computation time became long enough to delay my graduation.
2. Evidence of reproducibility: `targets` can confirm when everything is up to date, which is tangible evidence that the results are re-creatable from the current code and data. 
3. Data abstraction: each target is stored as a file in the `_targets`, but when you reference it from R, it looks and feels like an ordinary variable in an interactive R session. Coding up a pipeline feels natural and idiomatic, and there is no need to micromanage intermediate files.
3. Visualization. `targets::tar_visnetwork()` shows the high-level structure of the pipeline in an interactive directed acyclic graph, enhancing transparency and thus reproducibility.
